create_framework/routing.rst

```diff
@@ -43,13 +41,8 @@ One very important aspect of any website is the form of its URLs. Thanks to
 the URL map, we have decoupled the URL from the code that generates the
 associated response, but it is not yet flexible enough. For instance, we might
 want to support dynamic paths to allow embedding data directly into the URL
-instead of relying on a query string:
-
-    # Before
-    /hello?name=Fabien
-
-    # After
-    /hello/Fabien
+instead of relying on a query string (like ``/hello/Fabien`` instead of
```

Comment:
I'm not sure if the new version is much better than the old one. The repeated `instead of` is confusing to me.